src/components/ebay-button/README.md

```diff
@@ -10,15 +10,15 @@
 
 Name | Type | Stateful | Description
 --- | --- | --- | ---
-`priority` | String | No | "primary" / "secondary" (default) / "none"
-`size` | String | No | "small" or "large" (default: medium)
+`priority` | String | No | "primary" / "secondary" (default) / "delete" / "none"
+`size` | String | No | "large" (default: "none")
```

Comment:
Do we still support specifying `small`? Or is that class completely removed from Skin?